docs/ref/models/querysets.txt

```diff
@@ -2703,6 +2703,17 @@ When :setting:`USE_TZ` is ``True``, datetime fields are converted to the
 current time zone before filtering. This requires :ref:`time zone definitions
 in the database <database-time-zone-definitions>`.
 
+.. fieldlookup:: week
+
+``week``
```

Comment:
The documentation here could be made a bit more consistent with the docs for the other datetime lookups. E.g. in 8a4f017f there are also paragraphs like:
```
Example::

    ...

(No equivalent SQL code fragment...)

When :setting:`USE_TZ` is ``True``,...
```